ript.gemspec

```diff
@@ -7,8 +7,8 @@ Gem::Specification.new do |s|
   s.name        = "ript"
   s.version     = Ript::VERSION
   s.platform    = Gem::Platform::RUBY
-  s.authors     = [ "Lindsay Holmwood" ]
-  s.email       = [ "lindsay@bulletproof.net" ]
```

Comment:
I would recommend updating `AUTHORS.md` and explaining what `foundation@bulletproof.net` is.